src/FixedDataTable.js

```diff
@@ -1142,7 +1143,11 @@ class FixedDataTable extends React.Component {
       y = y > maxScrollY ? maxScrollY : y;
 
       //NOTE (jordan) This is a hacky workaround to prevent FDT from setting its internal state
-      if (onVerticalScroll ? onVerticalScroll(y) : true) {
+      if (
+        onVerticalScroll
+          ? onVerticalScroll(y, firstRowIndex, endRowIndex)
```

Comment:
Ah, this might be a problem. 
The `scrollPos` here refers to the future scroll position, but `firstRowIndex` and `endRowIndex` refers to current row indexes.
And without scrolling to the future scroll position, we can't figure out the future first and end row indexes...

This won't be seen at https://github.com/schrodinger/fixed-data-table-2/pull/596/files#diff-08c84b55091b201572f9bce34879c01b05a05b5a072dcb10d57f87f44d31f1b7R1264 though because at this point the table has already been scrolled.

@novakps , @mycrobe, @wcjordan , @somavara, @deshpsuy , rather than passing inconsistent row indexes I think we should just pass null here. But it still seems a bit weird.

Another way is to have a separate prop like `onViewportRowIndexesChange` be called the first and end row indexes have changed. In the future, we can have this also report back the first and end column indexes.

Reply:
Hmm, I guess it doesn't make much sense to pass null sometimes? Perhaps it would be indeed better to have diferent prop like `onViewportChange`.  Have you had the chance to discuss it?

Reply:
@manvydasu yep, we agreed that exposing `onViewportChange` is the better option.